source/community.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: has_complimentary
 title: '#teamSass'
-introduction: >
+introduction: |-
   Sass has an awesome community of designers and developers who love to spread
-  the word and help people out. Here we&rsquo;ve collected some resources.
-  Happy&nbsp;Styling!
+  the word and help people out. Here we've collected some resources.
+  Happy Styling!
```

Comment:
I don't think this is (currently) being parsed as markdown. Let me try adding a filter...

Reply:
I think that worked: https://github.com/oddbird/sass-site/commit/9c8bf022aa5e2a91e9f91fbc59f159137ab85163